cli/Sources/TuistServer/Client/ServerAuthenticationController.swift

```diff
@@ -7,7 +7,9 @@ import Mockable
 
 @Mockable
 public protocol ServerAuthenticationControlling: Sendable {
-    func authenticationToken(serverURL: URL) async throws -> AuthenticationToken?
+    @discardableResult func authenticationToken(serverURL: URL) async throws
+        -> AuthenticationToken?
```

Comment:
```suggestion
    func authenticationToken(serverURL: URL) async throws
        -> AuthenticationToken?
```

I don't think this method should have a discardable result. If any method should have it, it should be the `refreshToken` imho